distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -1129,7 +1129,7 @@ type DB_Table
        Create a constant column from a value.
     make_constant_column : Any -> DB_Column ! Illegal_Argument
     make_constant_column self value =
-        Feature.Column_Operations.if_supported_else_throw self.connection.dialect "make_constant_column" <|
+        Feature.Filter.if_supported_else_throw self.connection.dialect "make_constant_column" <|
```

Comment:
Do we make sure that this error is actually never exposed to the user? Since `make_constant_column` is a PRIVATE method, we probably don't want the user to see a message "make_constant_column is unsupported (etc.)" - instead we want to report the high-level method (i.e. `set`, `filter`) that has an unsupported feature.

Reply:
Maybe I should just remove this check then? It felt wrong flagging it under Filter.

Reply:
Yeah, I think we could remove it and just make sure the high-level functions have needed checks.

Reply:
Done